submissions/sprint_b/helpers.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+import random
+
+
+class FileManager:
+    """Handle local file system IO."""
+
+    @staticmethod
+    def get_extension(path):
+        """Get file extension from file path."""
+        return os.path.splitext(path)[-1][1:]
+
+    @staticmethod
+    def read_json(path, mode='r', *args, **kwargs):
+        with open(path, mode=mode, *args, **kwargs) as handle:
+            return json.load(handle)
+
+
+class Vocabulary:
+    """Standardize vocabulary representation from multiple sources."""
+
+    files = FileManager()
+
+    @classmethod
+    def from_file(cls, path, *args, **kwargs):
+        extension = cls.files.get_extension(path)
+        representation = cls.strategies(extension)(path, *args, **kwargs)
+        return representation[0]
+
+    @classmethod
+    def from_json(cls, path, fields=True, *args, **kwargs):
+        data = cls.files.read_json(path, *args, **kwargs)
+        if fields:
+            representation = (data, data.keys())
+        else:
+            representation = data
+        print(representation)
+        return representation
+
+    @classmethod
+    def strategies(cls, file_extension, intent='read'):
+        input_strategies = {'json': cls.from_json}
+        if intent is 'read':
+            return input_strategies[file_extension]
+
+
+class EpithetGenerator:
+    """Generates an epithet given three columns of data to choose from"""
+    @staticmethod
+    def generate_epithet(path):
```

Comment:
Your generate_epithet method should accept the quantity of epithets to generate and return a list of epithets. All of the logic for looping some quantity of times should be part of your generate_epithet method.


2. Define an EpithetGenerator class with methods to:
    1. select one random word from each column of the list.
    2. generate a quantity of epithets from a vocabulary file loaded from a path.